sdk/core/azure-core/src/main/java/com/azure/core/util/polling/DefaultPollingStrategy.java

```diff
@@ -76,10 +77,28 @@ public DefaultPollingStrategy(HttpPipeline httpPipeline, JsonSerializer serializ
      * @throws NullPointerException If {@code httpPipeline} is null.
      */
     public DefaultPollingStrategy(HttpPipeline httpPipeline, String endpoint, JsonSerializer serializer, Context context) {
+        this(new PollingStrategyOptions(httpPipeline)
+            .setEndpoint(endpoint)
+            .setSerializer(serializer)
+            .setContext(context));
+
+    }
+
+    /**
+     * Creates a chained polling strategy with 3 known polling strategies, {@link OperationResourcePollingStrategy},
+     * {@link LocationPollingStrategy}, and {@link StatusCheckPollingStrategy}, in this order, with a custom
+     * serializer.
+     *
+     * @param pollingStrategyOptions options to configure this polling strategy.
+     * @throws NullPointerException If {@code pollingStrategyOptions} is null or
+     * {@link PollingStrategyOptions#getHttpPipeline()} is null.
```

Comment:
Should this verify that? Or, if `HttpPipeline` is always required should it be a constructor parameter to the `PollingStrategyOptions`?

Reply:
It is a constructor param in `PollingStrategyOptions`.